src/cmake/modules/GUDHI_third_party_libraries.cmake

```diff
@@ -167,46 +168,37 @@ if (WITH_GUDHI_PYTHON)
     endif()
   endfunction( find_python_module )
 
-  # For modules that do not define module.__version__
-  function( find_python_module_no_version PYTHON_MODULE_NAME )
-    string(TOUPPER ${PYTHON_MODULE_NAME} PYTHON_MODULE_NAME_UP)
-    execute_process(
-            COMMAND ${Python_EXECUTABLE}  -c "import ${PYTHON_MODULE_NAME}"
-            RESULT_VARIABLE PYTHON_MODULE_RESULT
-            ERROR_VARIABLE PYTHON_MODULE_ERROR)
-    if(PYTHON_MODULE_RESULT EQUAL 0)
-      # Remove carriage return
-      message ("++ Python module ${PYTHON_MODULE_NAME} found")
-      set(${PYTHON_MODULE_NAME_UP}_FOUND TRUE PARENT_SCOPE)
-    else()
-      message ("PYTHON_MODULE_NAME = ${PYTHON_MODULE_NAME}
-       - PYTHON_MODULE_RESULT = ${PYTHON_MODULE_RESULT}
-       - PYTHON_MODULE_ERROR = ${PYTHON_MODULE_ERROR}")
-      set(${PYTHON_MODULE_NAME_UP}_FOUND FALSE PARENT_SCOPE)
-    endif()
-  endfunction( find_python_module_no_version )
-
   if( TARGET Python::Interpreter )
     find_python_module("cython")
     find_python_module("pytest")
     find_python_module("matplotlib")
     find_python_module("numpy")
     find_python_module("scipy")
     find_python_module("sphinx")
-    find_python_module("sklearn")
-    find_python_module("ot")
+    find_python_module("scikit-learn")
+    find_python_module("POT")
     find_python_module("pybind11")
     find_python_module("torch")
-    find_python_module("pykeops")
     find_python_module("eagerpy")
-    find_python_module_no_version("hnswlib")
+    find_python_module("hnswlib")
     find_python_module("tensorflow")
     find_python_module("sphinx_paramlinks")
     find_python_module("pydata_sphinx_theme")
-    find_python_module_no_version("sphinxcontrib.bibtex")
+    find_python_module("sphinxcontrib.bibtex")
     find_python_module("networkx")
   endif()
 
+  # Specific case for PyKeops that can be imported on Windows, but fails because it uses fcntl (not available on Windows)
+  # Also fcntl has no metadata, so find_python_module does not work
+  # "import fcntl" is about 1 sec. faster than "import pykeops"
+  execute_process(
+          COMMAND ${Python_EXECUTABLE}  -c "import fcntl"
+          RESULT_VARIABLE FCNTL_IMPORT_MODULE_RESULT
+          OUTPUT_VARIABLE FCNTL_IMPORT_MODULE_OUPUT)
+  if(FCNTL_IMPORT_MODULE_RESULT EQUAL 0)
```

Comment:
Is that better than `if (NOT WIN32)`?

Reply:
No, you are right. I wanted to do something smarter, like "the day fcntl will be available on windows, pykeops will work on windows", but that was not that smart...
`if (NOT WIN32)` on [b4694d9](https://github.com/GUDHI/gudhi-devel/pull/1135/commits/b4694d930d810c26c5aa5234dc059ce2f771595f)